releases.md

```diff
@@ -0,0 +1,100 @@
+# Tekton Pipeline Releases
+
+## Release Frequency
+
+Tekton Pipelines follows the Tekton community [release policy][release-policy]
+as follows:
+
+- Versions are numbered according to semantic versioning: `vX.Y.Z`
+- A new release is produced on a monthly basis
```

Comment:
Will this be the case after we release `V1`?

Reply:
Yes

Reply:
@pritidesai you mean a `v1` tag right ? 🙃